lib/plugins/package/lib/packageService.js

```diff
@@ -63,7 +66,11 @@ module.exports = {
 
     const exclude = this.getExcludes(funcPackageConfig.exclude);
     const include = this.getIncludes(funcPackageConfig.include);
-    const zipFileName = this.provider.naming.getFunctionArtifactName(functionName);
-    return this.zipDirectory(exclude, include, zipFileName);
+    const zipFileName = `${functionName}.zip`;
```

Comment:
Same here. Overkill, but maybe a `provider.naming` is a good place to store this value.